class-wc-calypso-bridge.php

```diff
@@ -286,7 +287,7 @@ public function decalypsoify_ecommerce_plan( $null, $object_id, $meta_key ) {
 	 */
 	public function add_ecommerce_plan_styles() {
 		$asset_path = self::$plugin_asset_path ? self::$plugin_asset_path : self::MU_PLUGIN_ASSET_PATH;
-		wp_enqueue_style( 'wp-calypso-bridge-ecommerce', $asset_path . 'assets/css/ecommerce.css', array(), WC_CALYPSO_BRIDGE_CURRENT_VERSION );
+		wp_enqueue_style( 'wp-calypso-bridge-ecommerce', get_home_url() . $asset_path . 'assets/css/ecommerce.css', array(), WC_CALYPSO_BRIDGE_CURRENT_VERSION );
```

Comment:
Why do we need to prefix this with `get_home_url()`? Was this broken?

Reply:
Hmm, I saw a strange HTML comment in the `<head>` about an invalid path ( 
`<!-- No Concat CSS wp-calypso-bridge-ecommerce => Invalid Path  -->`) that was solved by using the full URL here. 

Although I tested again without the full URL, it seems to be working! I might be caught up in a caching issue 🤔 Nice catch!